lib/db/seeds/mainnet.ts

```diff
@@ -14,22 +14,29 @@ const nodes = [
 const currencies = [
   { id: 'BTC', swapClient: SwapClientType.Lnd, decimalPlaces: 8 },
   { id: 'LTC', swapClient: SwapClientType.Lnd, decimalPlaces: 8 },
+  {
+    id: 'ETH',
+    swapClient: SwapClientType.Connext,
+    decimalPlaces: 18,
+    tokenAddress: '0x0000000000000000000000000000000000000000',
+  },
   {
     id: 'WETH',
-    swapClient: SwapClientType.Raiden,
+    swapClient: SwapClientType.Raiden, // needs to be configurable to support Connext
```

Comment:
I'm not quite sure if we want to have multiple swap clients per currency. Can you remind me what was the primary use case of having such functionality?

Reply:
The case here is that Connext and Raiden have an overlap of assets that they can swap. Meaning that if a user wants to swap either DAI or WETH it can be done either with Connext or Raiden. 

Alternatively you could duplicate the assets with different swapClients. Or just replace the Raiden client for DAI and  WETH

Reply:
> Or just replace the Raiden client for DAI and WETH

That is the ultimate goal. And definitely only one swap client per asset. Until then we'd like to keep raiden in. Proposal how to do that: Leave WETH & DAI to Raiden. Give ETH & USDT to connext. Once everything is integrated, working and tested we'll also switch WETH & DAI to connext.

Reply:
I'll take a look at better handling the option of more than 1 client for a given currency in a separate PR.

Reply:
Added USDT and assigned it SwapClientType.Connext